tools/languagelocalizer/languageLocalizer.js

```diff
@@ -128,36 +38,41 @@ const LanguageLocalizer = {
 
   async init() {
     program.version('0.0.1')
+    program.option("-c, --check", `
+      Check only, will fail if user input would be required to generate 
+      the file.
+    `);
 
-    program.option(
-        '-p, --path <mozillavpn>',
-        `config file. Default: ${DEFAULT_MOZILLAVPN}`, DEFAULT_MOZILLAVPN);
     program.action(async () => await this.run(program.opts()));
 
     program.parseAsync(process.argv);
   },
 
   async run(options) {
-    this.execClient(options.path);
-    await this.connectToClient();
-
+    this.options = options;
+    if(options.check){
+      console.log("Running Checks Only");
+    }
     this.wbk = WBK({
       instance: 'https://www.wikidata.org',
       sparqlEndpoint: 'https://query.wikidata.org/sparql',
     })
+    const __filename = fileURLToPath(import.meta.url);
+    const __dirname = path.dirname(__filename);
```

Comment:
Just curious, why is the double underscore prefix naming pattern being used here?